content/en/blog/_posts/2025-10-05-gateway-api-1.4/index.md

```diff
@@ -0,0 +1,509 @@
+---
+layout: blog
+title: "Gateway API 1.4: New Features"
+date: 2025-11-06T09:00:00-08:00 # suggested
+draft: true
+slug: gateway-api-v1-4
+author: >
+    [Beka Modebadze (Google)](https://github.com/bexxmodd),
+    [Gateway API Contributors](contributors.md)
+---
+
+![Gateway API logo](gateway-api-logo.svg)
+
+Ready to rock your Kubernetes networking? The Kubernetes SIG Network community presented the General Availability (GA) release of Gateway API (v1.4.0)! Released on October 6, 2025, version 1.4.0 reinforces the path for modern, expressive, and extensible service networking in Kubernetes.
+
+Gateway API v1.4.0 brings three new features to the _Standard channel_
+(Gateway API's GA release channel):
+
+* **BackendTLSPolicy for TLS between gateways and backends**
+* **`supportedFeatures` in GatewayClass status**
+* **Named rules for Routes**
+
+and introduces three new experimental features:
+
+* **Mesh resource for service mesh configuration**
+* **Default gateways** to ease configuration burden**
+* **`externalAuth` filter for HTTPRoute**
+
+## Graduations to Standard Channel
+
+### Backend TLS Policy
+
+Leads: [Candace Holman](https://github.com/candita), [Norwin Schnyder](https://github.com/snorwin), [Katarzyna Łach](https://github.com/kl52752)
+
+GEP-1897: [BackendTLSPolicy](https://github.com/kubernetes-sigs/gateway-api/issues/1897)
+
+[BackendTLSPolicy](https://gateway-api.sigs.k8s.io/api-types/backendtlspolicy) is a new Gateway API type for specifying the TLS configuration
+of the connection from the Gateway to backend pod(s).
+This is also known as _backend TLS termination_, and provides information to the Gateway on how to connect
+to a backend pod that has its own certificate.  Prior to the introduction of BackendTLSPolicy, there was no API specification
+that allowed encrypted traffic on the hop from Gateway to backend.
+
+The BackendTLSPolicy `validation` configuration requires either a `hostname` or one or more `subjectAltNames` (SAN).
```

Comment:
BackendTLSPolicy always requires a `hostname`, which is used as the SNI when connecting to the backend. Additionally, it is used for certificate validation if, and only if, no `subjectAltNames` are configured.

https://github.com/kubernetes-sigs/gateway-api/blob/main/apis/v1/backendtlspolicy_types.go#L194